dspace/config/dspace.cfg

```diff
@@ -1008,6 +1008,14 @@ metadata.hide.person.email = true
 # Disabled by default.  To enable, uncomment the next line
 # metadata.privacy.dc.description.provenance = true
 
+# Enable/disable bitstream provenance tracking (default: true)
+# When disabled, also disables provenance for submission/workflow bitstream operations
+bitstream.provenance.enabled = true
```

Comment:
Minor thing, I'd recommend renaming these new configs to *start* with `provenance`, since that's the feature they relate to.

So, instead call them:
```
provenance.bitstream.enabled = true
provenance.bitstream.bundles.exclude = THUMBNAIL,TEXT
```

This would also allow us to more easily add more provenance configs in the future and relate them all together.

Reply:
agreed & done!